deeplake/api/tests/test_linking.py

```diff
@@ -10,7 +10,7 @@ def test_linking(memory_ds):
     with ds:
         ds.create_tensor("x")
         ds.create_tensor("y")
-        ds._link_tensors("x", "y", "append_test")
+        ds._link_tensors("x", "y", "extend_test")
```

Comment:
We are switching to extend instead of append for linked tensors to benefit from the optimizations.